package.json

```diff
@@ -5,31 +5,32 @@
   "version": "1.0.0",
   "type": "module",
   "scripts": {
-    "build": "remix build",
-    "dev": "remix dev --manual -c \"netlify dev\"",
-    "preview": "netlify serve",
+    "build": "remix vite:build",
```

Comment:
We can't use `shopify hydrogen build` here because of https://github.com/Shopify/hydrogen/issues/2497.